Ginger/GingerCore/Actions/ActScript.cs

```diff
@@ -141,7 +141,7 @@ protected void AddData(string outLine)
         }
         public override void Execute()
         {
-            if (ScriptName == null)
+            if (ScriptName == null && ScriptCommand == null)
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Change this condition so that it does not always evaluate to 'false'; some subsequent code is never executed.](https://app.codacy.com/manual/Ginger-Automation/Ginger/pullRequest?prid=4970873)

Reply:
**if (ScriptName == null)** - this verification should be executed only in case of "Scrip Action" is selected as "Script"  in UI.  This verification shouldn't be executed in case of "FreeCommand". How can we add this to verification?  
![image](https://user-images.githubusercontent.com/59820100/74148264-bac54e00-4c0d-11ea-8a9d-cc8c8d65e29c.png)